docs/source/markdown/options/volume.md

```diff
@@ -156,6 +156,12 @@ To recursively mount a volume and all of its submounts into a
 used, and submounts of the source directory will not be mounted into the
 <<container|pod>>.
 
+Mounting the volume with the **copy** options tells podman to copy content from
```

Comment:
I think this should be `option`, singular, not plural. LGTM other than this and the "not" question below.